packages/debug/src/browser/view/debug-toolbar-widget.tsx

```diff
@@ -77,6 +81,29 @@ export class DebugToolBar extends ReactWidget {
             {this.renderStart()}
         </React.Fragment>;
     }
+
+    protected renderContributedCommands(): React.ReactNode {
+        return this.menuModelRegistry
+            .getMenu(DebugToolBar.MENU)
+            .children.filter(node => node instanceof CompositeMenuNode)
```

Comment:
Use `CompoundMenuNode.is` here rather than the `instanceof` check, and add a `when` clause check here, as well, since submenus can have their own `when` requirements.